core/src/main/java/bisq/core/dao/governance/proposal/param/ChangeParamValidator.java

```diff
@@ -214,24 +222,64 @@ private void validateParamValue(Param param, String inputValue, int chainHeight)
             case PHASE_VOTE_REVEAL:
             case PHASE_BREAK3:
                 if (isMainnet)
-                    checkArgument(newValueAsBlock >= 6, "The break must have at least 6 blocks");
+                    checkArgument(inputValueAsBlock >= 6, "The break must have at least 6 blocks");
                 break;
             case PHASE_RESULT:
                 if (isMainnet)
-                    checkArgument(newValueAsBlock >= 1, "The break must have at least 1 block");
+                    checkArgument(inputValueAsBlock >= 1, "The break must have at least 1 block");
                 break;
+        }
+    }
+
+    private void validateAddressValue(String currentParamValue, String inputValue) {
```

Comment:
Should add `throws AddressFormatException` to this method signature

Reply:
AddressFormatException is a RuntimeException. But agree whould be more clean to add it. Will consider to wrap it into ValidationException.

Reply:
Used ParamValidationException in last commit.